engineapi/engineapi/models.py

```diff
@@ -155,14 +157,88 @@ class DropperClaimant(Base):  # type: ignore
     )
 
 
+class CallRequestType(Base):  # type: ignore
+    """
+    CallRequestType contains versions of call requests like:
+    raw or dropper-v0.2.0.
+    """
+
+    __tablename__ = "call_request_types"
+
+    id = Column(
+        UUID(as_uuid=True),
+        primary_key=True,
+        default=uuid.uuid4,
+        unique=True,
+    )
+    request_type = Column(VARCHAR(128), nullable=False, unique=True, index=True)
+    description = Column(String, nullable=True)
+    required_params = Column(ARRAY(String), nullable=False)
```

Comment:
@kompotkot and I discussed this.

My feeling:
- Better to use JSON Schema for validation - https://json-schema.org/. Other request types (e.g. Ethereal mint) might have nested JSON structures inside the parameters object and `required_params` will quickly become obsolete as it only checks keys.
- Better to store schemas in code (in git). This way, all schema changes that make it to production can be reviewed and audited.
- We don't need a UUID primary key - we can just use the `request_type` as a primary key for this table. This will remove the need to read UUIDs from database in order to insert data into `call_requests`.

Reply:
JSON schema validation - create issue doesn't have to be implemented in this PR. We can use existing validation for now.